content/doc/tutorials/using-jenkinsfile-runner-github-action-to-build-jenkins-pipeline.adoc

```diff
@@ -0,0 +1,308 @@
+        }
+    }
+}
+----
+
+=== Create a plugin list file
+
+In this section, you will need to create a plugin list file which specifies which plugins you will need to install in the ephemeral Jenkins instance. 
+You can check link:https://github.com/jenkinsci/plugin-installation-manager-tool#plugin-input-format[the valid plugin input format]. 
+As we don’t have to install any extra plugins, you can create an empty plugin list file called “plugins.txt”.
+
+=== Create a workflow definition in the container
+
+In this section, you’ll learn how to create a workflow definition so you can run Jenkins pipeline in the GitHub Actions.
+
+* In your local GitHub repository, prepare a workflow definition file ci.yml (the file name doesn’t matter but it should be a YAML file) under the “.github/workflows” directory.
+* Create the name of your workflow and the event that triggers your workflow. 
+You can check link:https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#available-events[the available events] that trigger workflows. 
+The “push” event will trigger the workflow when you push a commit or tag.
+[source,yaml]
+----
+name: Java CI
+on: [push]
+----
+* Use a Ubuntu runner for the job.
+[source,yaml]
+----
+jobs:
+  job-name:
+    runs-on: ubuntu-latest
+----
+* (Optional) If you want to use `jfr-container-action` later, you need to declare using the `ghcr.io/jenkinsci/jenkinsfile-runner:master` or any image extending it. 
+If you choose to use `jfr-static-image-action`, you can skip this step.
+[source,yaml]
+----
+jobs:
+  job-name:
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/jenkinsci/jenkinsfile-runner:master
+----
+* Call the `actions/checkout@v2` to pull your codes into the runner. 
+“Call” means “uses” in the workflow definition specifically. 
+You can check the details about the “uses” keyword.
+[source,yaml]
+----
+- uses: actions/checkout@v2
+----
+* Call the Jenkinsfile-runner actions.
+** If you use `jfr-container-action`, you need to call `jenkinsci/jfr-container-action@master` and give necessary inputs.
+[source,yaml]
+----
+uses:
+  jenkinsci/jfr-container-action@master
+with:
+  command: run
+  jenkinsfile: Jenkinsfile
+  pluginstxt: plugins.txt
+----
+** If you use `jfr-static-image-action`, you need to call `jenkinsci/jfr-static-image-action@master` and give necessary inputs.
+[source,yaml]
+----
+uses:
+  jenkinsci/jfr-static-image-action@master
+with:
+  command: run
+  jenkinsfile: Jenkinsfile
+  pluginstxt: plugins.txt
+----
+
+You can check the full example workflow definition for link:http://jfr-static-image-action[jfr-container-action] and link:http://jfr-static-image-action[jfr-static-image-action]. 
+There is another powerful GitHub Action list called Jenkinsfile Runner Runtime Actions. 
+For these GitHub Actions, you can run them in any runners, which are Windows, Linux and macOS. 
+You can check their step by step usage link:http://jfr-static-image-action[here].
```

Comment:
```suggestion
There is another powerful GitHub Action list called Jenkinsfile Runner Runtime Actions. 
For these GitHub Actions, you can run them in any runners, which are Windows, Linux and macOS. 
You can check their step by step usage link:http://jfr-static-image-action[here].
```
Again, the link does not lead anywhere/work properly, so it should either be removed, or the correct link needs to be inserted.
This also seems like related information, so is it something that users should be aware of in this tutorial, or is it something better left towards the end for "continued reading"? Introducing the concept and immediately moving on is not the best experience.

Reply:
Thanks for pointing out this problem. Let me update it later.

Reply:
I just updated the URLs in the latest commit.

Yes, it is related information. But they are other Jenkinsfile Runner GitHub Actions which have the same functions as the two GitHub Actions explained above but have different implementations (so that they can be used in different runners). I think developers only need to understand one method about how to use this tool in this introduction tutorial. If they want to learn more, they can move on to the official guide. I did think about using a "continued reading" in the end to explain these GitHub Actions but I think they are too short to be a new part :)

What's your opinion?

Reply:
For this issue, I temporarily mentioned the other two Jenkinsfile Runner GitHub Actions here and recommended the developers to learn more in the links of Wrapping up section. Feel free to reopen this issue.